selfservice/flow/login/organizations.go

```diff
@@ -0,0 +1,42 @@
+// Copyright © 2025 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+package login
+
+import (
+	"net/http"
+
+	"github.com/ory/kratos/selfservice/flow"
+
+	"github.com/ory/kratos/identity"
+	"github.com/ory/kratos/session"
+)
+
+var organizationFilter = []StrategyFilter{func(s Strategy) bool {
+	a, b := s.(flow.OrganizationImplementor)
+	return b && a.SupportsOrganizations()
+}}
+
+func PrepareOrganizations(r *http.Request, f *Flow, sess *session.Session) []StrategyFilter {
+	if f.RequestedAAL != identity.AuthenticatorAssuranceLevel1 {
+		return []StrategyFilter{}
+	}
+
+	if f.OrganizationID.Valid {
```

Comment:
Is it really safe to store the organization ID in the flow and trust it? What if I initialize the flow with one organization, but then update the identifier so I use a different one?
As far as I can see, this behavior is new.